lib/sidekiq_unique_jobs/locksmith.rb

```diff
@@ -241,7 +241,7 @@ def enqueue(conn)
     def primed_async(conn, wait = nil, &block)
       primed_jid = Concurrent::Promises
                    .future(conn) { |red_con| pop_queued(red_con, wait) }
-                   .value(add_drift(wait || config.timeout))
+                   .value(add_drift(wait || config.timeout || 0))
```

Comment:
Code looks good, but any idea why `config.timeout` would be nil?

Reply:
That is actually expected, the readme suggests it in the example: https://github.com/mhenrixon/sidekiq-unique-jobs#global-configuration

```
config.lock_timeout    = nil   # turn off lock timeout
```